d2/src/main/java/com/linkedin/d2/jmx/ZooKeeperAnnouncerJmx.java

```diff
@@ -206,4 +206,9 @@ public int getWeightDecimalPlacesBreachedCount()
   {
     return _announcer.getWeightDecimalPlacesBreachedCount();
   }
+
+  @Override
+  public int getServerAnnounceMode() {
+    return _announcer.getServerAnnounceMode().ordinal();
```

Comment:
For JMX debug, Maybe `toString()` is better ?

Reply:
this is for ingraph, which can't display string, please see container PR.

Reply:
once we have ingraph, which has detailed comment on which number means which mode, it will be a lot easier to use than calling jmx. Why we need to use jmx to debug (this class is needed just for jmxIngraph to work)?

Reply:
> once we have ingraph, which has detailed comment on which number means which mode,
Yeah. That makes sense. For sure, we could check ingraph to learn the mode.